api/api_autogen/library/defaults/Battery_FuelCellCommercial.json

```diff
@@ -169,6 +169,8 @@
 		},
 		"PriceSignal": {
 		},
+		"Revenue": {
```

Comment:
Why is this added to every file?

Reply:
That comes from https://github.com/NREL/ssc/pull/787/files - which is an ssc vartable input group for revenue forecasting.